WalletWasabi.Gui/Controls/WalletExplorer/SendTabViewModel.cs

```diff
@@ -1171,6 +1175,15 @@ public override void OnOpen()
 				.ObserveOn(RxApp.MainThreadScheduler)
 				.Subscribe(_ => SetFeesAndTexts());
 
+			Global.UiConfig
+				.WhenAnyValue(x => x.FeeDisplayFormat)
+				.ObserveOn(RxApp.MainThreadScheduler)
+				.Subscribe(x =>
+			{
+				FeeDisplayFormat = (FeeDisplayFormat)(Enum.ToObject(typeof(FeeDisplayFormat), x) ?? FeeDisplayFormat.SatoshiPerByte);
+				SetFeesAndTexts();
```

Comment:
`SetFeesAndTexts();` is a bad practice. We should not double down on this for the sake of this PR.